Formula/dynamodb-local.rb

```diff
@@ -67,4 +67,8 @@ def plist; <<-EOS.undent
     </plist>
     EOS
   end
+
+  test do
+    system "#{bin}/dynamodb-local", "-help"
```

Comment:
> -help means Java is able to run the JAR. If the JAR had -version I would do that like the Elixir formula does, but it doesn't have that.

Yes I know, and `-version` wouldn't be too thrilling a test either. Just wondering if you can squeeze some actual dynamodb usage into the test. Maybe insert something into the database then retrieve it? I'm not a user of this software and don't wish to learn to use every piece of software in Homebrew, so unfortunately I can't make suggestions.

If you tell me for sure that a more substantial test is impractical, I've no problem sticking with this.